Sources/AuthFoundation/Network/APIClient.swift

```diff
@@ -361,3 +346,5 @@ let defaultJSONDecoder: JSONDecoder = {
     decoder.keyDecodingStrategy = .convertFromSnakeCase
     return decoder
 }()
+
+var _APIClientRetryDelayTimeIntervalToNanoseconds: Double = 1_000_000_000
```

Comment:
Why not use one of the Dispatch intervals like `.seconds(1)` to simplify?

Reply:
That requires iOS 16+, so the SDK still uses the older `nanoseconds` notation.